stustapay/tse/diebold_nixdorf_usb/tse_verifier.py

```diff
@@ -0,0 +1,108 @@
+import ecdsa
+from hashlib import sha384, sha256
+import base64
+from dateutil import parser
+from datetime import timezone
+import pprint
+from asn1crypto.core import Integer, ObjectIdentifier, Sequence, OctetString, PrintableString, Any
```

Comment:
asn1crypto as dependency?

Reply:
do you want to build DER objects by hand?

Reply:
it's missing in the pyproject.toml :)